src/dss_cell_group.hpp

```diff
@@ -56,25 +49,25 @@ class dss_cell_group : public cell_group {
     {}
 
     void advance(time_type tfinal, time_type dt) override {
-        for (auto cell_idx: util::make_span(0, not_emit_it_.size())) {
+        for (auto lid: util::make_span(0, not_emit_it_.size())) {
```

Comment:
`lid` is used as an index into other vectors within the look body though. Functional solution would be to have a `zip`-like view…

That said, not convinced `lid` is really a good name for a loop index; elsewhere, we tend to use `lid` to refer to the local index or identifier of a type of thing within one cell. Sometimes `i` for a loop index is all we need.